functions/common/lib.esm/addresses.json

```diff
@@ -1,6 +1,6 @@
 {
     "goerli": {
-        "hexlink": "0xbad6a7948a1d3031ee7236d0180b6271fa569148",
+        "hexlink": "0x9f81B7588A4Bd2b0e5893f12666c6475a63022ae",
```

Comment:
Please use the latest one 0x4c552dC72756A690883f9e8955B231c43c4E598e. We have to re-deploy it due to https://github.com/HexlinkOfficial/hexlink-contracts-evm/pull/5